README.md

```diff
@@ -13,6 +13,8 @@ Warning: PHP versions 7.2 and older are deprecated, and will cause problems, unr
 
 Change Log
 ----------
+* 3.11, release 2 2022.05.03
+  * Fixed sharing cart looking at invisible modules, where error (invalid id for course module) would occur.
```

Comment:
Why not merge this change into ``3.11, release 1 2022.05.03``, since it hadn't yet been merge into the upstream repository and it got change in a short time period?

Reply:
Changes added